StellarApp/Controllers/SolarViewController.swift

```diff
@@ -20,20 +25,57 @@ class SolarViewController: UIViewController, ARSCNViewDelegate {
         sceneView.delegate = self
         
         sceneView.showsStatistics = true
-
-        sceneView.scene.rootNode.addChildNode(sunNode)
-        Planet.getPlanets().forEach { sunNode.addChildNode($0) }
         
+        sceneView.scene.rootNode.addChildNode(sunNode)
         
+        PlanetsAction.getPlanets().forEach { sunNode.addChildNode($0) }
         
 
-        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapped))
-        let pinchGestureRecognizer = UIPinchGestureRecognizer(target: self, action: #selector(pinch))
-        sceneView.addGestureRecognizer(tapGestureRecognizer)
-        sceneView.addGestureRecognizer(pinchGestureRecognizer)
-
     }
     
+    @IBAction func switchAction(_ sender: UISwitch) {
+        if playAnimation == .regular {
+            playAnimation = .animation
+        } else {
+            playAnimation = .regular
+        }
+        
+        switch playAnimation {
+        case .regular:
+            
+            sceneView.scene.rootNode.enumerateChildNodes { (node, stop ) in
+                node.removeFromParentNode()
+            }
+            let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapped))
+            let pinchGestureRecognizer = UIPinchGestureRecognizer(target: self, action: #selector(pinch))
+            let rotateGestureRecognizer = UIRotationGestureRecognizer(target: self, action: #selector(rotate))
+            sceneView.addGestureRecognizer(tapGestureRecognizer)
+            sceneView.addGestureRecognizer(pinchGestureRecognizer)
+            sceneView.addGestureRecognizer(rotateGestureRecognizer)
+            
+            sceneView.scene.rootNode.addChildNode(sunNode)
+            Planet.getPlanets().forEach { sunNode.addChildNode($0) }
+            sceneView.isUserInteractionEnabled = true
+            
+        case .animation:
+            sceneView.scene.rootNode.enumerateChildNodes { (node, stop ) in
+                node.removeFromParentNode()
+            }
+            let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapped))
+            let pinchGestureRecognizer = UIPinchGestureRecognizer(target: self, action: #selector(pinch))
+            let rotateGestureRecognizer = UIRotationGestureRecognizer(target: self, action: #selector(rotate))
+            sceneView.removeGestureRecognizer(tapGestureRecognizer)
+            sceneView.removeGestureRecognizer(pinchGestureRecognizer)
+            sceneView.removeGestureRecognizer(rotateGestureRecognizer)
```

Comment:
does rotating and panning have to be tied together? Is it possible that you may want to turn off rotation but leave on scaling/tapping? Seems like rotation is the only actual animation.